OpenIPSL/Electrical/Controls/PSSE/TG/BaseClasses/GGOV1/Turbine.mo

```diff
@@ -71,6 +69,14 @@ model Turbine "GE General GGOV1 and GGOV1DU Turbine Model"
     Placement(transformation(extent = {{-160, 0}, {-140, 20}})));
   Modelica.Blocks.Nonlinear.Limiter limiter(limitsAtInit = true, uMax = Vmax, uMin = Vmin) annotation (
     Placement(transformation(extent = {{-48, -50}, {-28, -30}})));
+  Modelica.Blocks.Nonlinear.FixedDelay fixedDelay(delayTime=Teng) if
+       delayType == Types.DelayType.FixedDelay "Use fixed delay model"
+    annotation(Placement(transformation(extent={{108,-32},{120,-20}})));
+  Modelica.Blocks.Nonlinear.PadeDelay padeDelay(
+    delayTime=C.eps,
```

Comment:
```suggestion
    delayTime=Teng,
```

Reply:
The reason I changed from `Teng` to `C.eps` here is that you would normally only use PadeDelay in case you want to run linearisation. And for that, the documentation says the default is set to `C.eps`. Leaving it set to `Teng` would be wrong here.

Reply:
Aah, I see. I should've taken a better look into PadeDelay block.  But it makes sense to me now, I guess.